frontend/views/containers/forms/FormPassword.vue

```diff
@@ -20,10 +22,9 @@ label.field
       i(:class='isLock ? "icon-eye" : "icon-eye-slash"')
 
   i18n.error(
-    tag='p'
-    v-show='v[name].$error'
+    v-show='vForm[name].$error'
```

Comment:
@taoeffect I tried to use v-error in this component, but couldn't make it work because the form key is dynamic and v-error always look for `this.$v.form.` which doesn't exist in this component (it's `this.vForm`).

Reply:
Have you tried passing down `$v` instead of `$v.form`?

Reply:
this wouldn't fix it. the problem is not the prop, is the v-error because it looks for `this.$v.form` when it should look for `this.vForm.form`.